script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_5B.txt

```diff
@@ -98,55 +98,55 @@ All I want right now is to be left alone in my room, take off my glasses, close
 <Page20>
 "―――――――"
-@gMy throat is aching.
-@gI haven't been able to breathe properly for a while now.
+@gMy THROAT is ACHING.//@g　ノドがアツイ。katakana (candidate for font/italics change)
+@gI haven't been ABLE to BREATHE for a WHILE now.//@g　サッキからイキがぜんぜんデキナイ。 (candidate for font/italics change)
 @gWhat's wrong with that? It's only natural.
-@gAfter witnessing such a ■, it would be rude for a creature to not get excited.
+@gAfter witnessing such a ■, it would be rude for a creature not to get excited.
 <Page21>
 @gI take something out of my bag.
 @gI put it in my pocket and start to walk.
 @gI feel the touch of iron on my fingertips.
 @gI am truly blessed.
 @gThis tool is just perfect.
 <Page22>
-@gHow wonderful. I'm going to come. What a sensual encounter.
+@gHow wonderful. I'm going to climax. What a sensual rendezvous.//@g　すばらしい。いきそうだ。なんて官能的な逢瀬。
 @gI must keep a good distance.
-@gDon't be noticed.
-@gDon't let anyone around us get in the way.
+@gSo as not to be noticed.//@g　気付かれないように、
+@gSo as not to arouse the suspicions of the nuisances that surround us.//@g　周りの邪魔者に気取られないように。
 @gThat woman and I are complete strangers.
-@gSo I have to follow her scent as naturally as possible.
+@gSo, as naturally as possible, I have to follow her scent.//@g　だから出来るだけ自然に、匂いを辿っていかなければ。
 <Page23>
 @gThe woman enters an apartment complex.
 @gAn automatic glass door. The security is perfect.
-@gNo one can even enter the hall unless they're residents.
-@g I go around to the emergency exit, take off my jacket and leave it somewhere hidden. I cut. And go inside.
+@gNo one can even enter the hall unless they're a resident.
+@g I go around to the emergency exit, take off my jacket, and leave it somewhere hidden. I cut. And I go inside.
 <Page24>
 @gThere's no one in the hall.
-@gI look at the elevator. The light has stopped at the top floor.
+@gI look at the elevator. The light stopped at the top floor.
 <Page25>
 "――――――"
 @gI press the call button. I go inside.
 <Page26>
 @gAt that moment, I'm overwhelmed by her remaining presence.
 @gThe lingering scent of that woman makes my mouth twitch upwards.
 <Page27>
-@gI'm excited.
-@gIn the tight confines of this closed room, I clutch the object in my pocket.
+@gI'm trembling with excitement.//@g　わくわくする。
+@gIn the tight confines of this locked room, I clutch the object in my pocket.//@g　狭い密室の中で、ポケットの中の得物を握り締めた。
 @gShe's right here, near me.
 @gNot much longer, I can finally ■ her.
 <Page28>
 @gI get out of the elevator.
 @gI sense that there is no one in the hall.
 @gI get out of the closed room. I head for the hallway. Not a soul in sight.
 @gI can't think.
-@gThere is only one word that pops into my head.
+@gMy mind can only bring forth a single word://@g　ある一つの単語しか、俺の脳味噌は生み出さない。
 <Page29>
-@gWoman.
+@gWoman.//@g　女を。I think the "一つの単語" is the verb he's going to do to the woman, not the woman. With that said, I don't actually know how else to deal with this.
```

Comment:
If we're willing to use the box symbol,

>■ that woman.
To ■ that woman.
At this rate, I'm going to――――――

Reply:
Good idea!